src/pages/MyList.js

```diff
@@ -1,23 +1,36 @@
-import ClassCard from "../components/ClassCard";
+import ClassCard from "../components/ClassCard/ClassCard";
 import { useEffect, useState } from "react";
 
 export default function MyList() {
   const [savedClasses, setSavedClasses] = useState([]);
 
+  function handleRemoveClass(id) {
+    const newSavedClasses = savedClasses.filter(
+      (savedClass) => savedClass.id !== id
+    );
+    setSavedClasses(newSavedClasses);
+  }
+
```

Comment:
Super awesome remove function. Very good job here. This looks very professional.